tests/test_suite.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 
 from giskard import Model, Suite
-from giskard.core.suite import single_binary_result
+from giskard.core.suite import single_binary_result, SuiteResult
```

Comment:
Small details but we have a hook to sort imports in order to be able to solve conflicts efficiently 

```suggestion
from giskard.core.suite import SuiteResult, single_binary_result
```